src/Main.java

```diff
@@ -1,4 +1,53 @@
 public class Main {
+
     public static void main(String[] args) {
+        findLeapYear(2000);
+        settingParameters(2022, 1);
+        calculationDay(95);
+    }
+//task.1
+    public static void findLeapYear(int year) {
+        if (year >= 1584) {
+            if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)) {
+                System.out.println(year + " is a leap year.");
+            } else {
+                System.out.println(year + " is not a leap year.");
+            }
+        } else {
+            System.out.println(year + " is not a leap year, because this concept was introduced in 1584.");
+        }
```

Comment:
это всё можно было сделать покороче и попроще:
```suggestion
        if (year >= 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
            System.out.println(year + " is a leap year.");
        } else {
            System.out.println(year + " is not a leap year");
        }
```